src/options.h

```diff
@@ -1623,7 +1623,8 @@ indent_paren_nl;
 // 0: Indent to body level (default)
 // 1: Align under the open parenthesis
 // 2: Indent to the brace level
-extern BoundedOption<unsigned, 0, 2>
+// -1: Preserve original indentation
+extern BoundedOption<signed, -1, 2>
```

Comment:
It would be fine to align the colons

Reply:
I have added spaces before the positive numbers so that the colons are aligned.